### PR TITLE
fixed from PVS-Studio

### PR DESCRIPTION
**Pull request by ip-gpu**
I'm a member of the Pinguem.ru competition on finding errors in open source projects. A bug, found using PVS-Studio. 

Your application has minimum supported OS is Windows 10, because of IVirtualDesktopManager interface usage.
You obtain this interface and has wrong check of result. This result is not boolean, therefore you doesn't get output error string even if you run this app not under Win10.
And there is no exit from app if you don't get success result of obtaining of interface.
You should apply it and your app will not crashes under other versions of Windows OS.

[V545](https://www.viva64.com/en/w/v545/) Such conditional expression of 'if' statement is incorrect for the HRESULT type value 'dpi_success'. The SUCCEEDED or FAILED macro should be used instead. main.cpp 976
[V545](https://www.viva64.com/en/w/v545/) Such conditional expression of 'if' statement is incorrect for the HRESULT type value 'desktop_success'. The SUCCEEDED or FAILED macro should be used instead. main.cpp 1015

### CODE REVIEW

**Comment by sylveon** [comment]
I don't think we should return, as the program can work fine without them. The failed macro is already used in the store branch iirc, which will possibly be the next release when I find time to wrap things up.

**Comment by ip-gpu** [comment]
The program works fine without exit only on Windows 10. But if you run it on Windows 7/XP, etc., it will crashes.

**Comment by ChriscomIT** [comment]
just fix it or leave it ... this project is really low lvl.excuses all the time no solutions at all.ur fu... up this prog its depreciated!!! why even covered on winfuture.de??dont get it

**Comment by ethanhs** [comment]
@ChriscomIT insulting the developers has no place here. Please refrain from doing so in the future or I will have no other option but to ban you.

The developers and contributors to TranslucentTB have spent time *for free* so that you can enjoy this program. If you don't like it, don't use it. Many others have liked it, as demonstrated by 121 thousand downloads and 1100 stars on Github (including yours, I might note).

Aside from your baseless accusations of the developers making excuses, I want to focus on your accusation that, as you put it, there are "no solutions at all". I will say this: we do this in our spare time. I am a full time university student, this started out as a hack I felt others may like, so I shared online. I no longer have time to work on it. There have been many generous contributors and core developers who have worked on it since.

It is important to note that none of the contributors have any obligation of contributing further, and we owe nothing to you. You are not compensating us in any way, so demanding us to work for your benefit is ridiculous. @sylveon has been generous to donate time from his life working on quite a few bug fixes and improvements. You can see his work in the `store` branch and what you'll see is dozens of commits of improvements and bug fixes.

So in short I'm not surprised you don't get it, as you seem not to know what you are talking about.

**Comment by Gymnasiast** [comment]
@ChriscomIT I'd ask for a fucking refund. Who do you think you are? You get a program for free and you think that entitles you to talk like that? Just fuck off, or fucking fix it yourself.

**Comment by ethanhs** [comment]
@ip-gpu apologies for cluttering this PR. I appreciate your working on this, however it seems @sylveon has already implemented it in the store branch, which will very likely be the next version released. Therefore I am closing this, as it is essentially resolved.

**Comment by ChriscomIT** [comment]
I just expressed my personal opinion of the state of the program/ the development over time. I did not insult anyone. Its just my opinion. BTW I hoped for an update since a long time but there is nothing so the prog is useless I deleted it without hope it will ever get better and the devs are pretty lazy in my opinion. Feel free to have another opinion.

**Comment by sylveon** [comment]
I've not taken it personally, I've seen much worse. However, both of your messages where rude and undesired. The "fu" in your first message could also be read as "fuck you", which we all probably know is an insult and shouldn't take place in an open-source project. I'll proceed to lock this thread to avoid more degeneration.
